lib/subjects/selector.rb

```diff
@@ -123,9 +123,12 @@ def missing_subjects_error
       MissingSubjects.new("No data available for selection")
     end
 
-    def subjects_page_size
-      page_size = params[:page_size] ? params[:page_size].to_i : 10
-      params.merge!(page_size: page_size)
+    def subjects_page_size(default_page_size=10)
```

Comment:
Doesn’t look like this method uses its default value since it’s been added to the workflow model.

Reply:
good catch - i've ensured that value can flow to the `workflow.selector_page_size` method now.